store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreITs.java

```diff
@@ -30,7 +29,6 @@
 import static uk.gov.gchq.gaffer.integration.junit.extensions.IntegrationTestSuiteExtension.INIT_CLASS;
 
 @ConfigurationParameter(key = INIT_CLASS, value = "uk.gov.gchq.gaffer.federatedstore.integration.FederatedStoreITs")
-@IncludeClassNamePatterns(".*IT")
```

Comment:
Why is this being removed? It was added here by @GCHQDeveloper314 [quite recently](https://github.com/gchq/Gaffer/blame/aeddaac462ee5ecbc3dfa00040a081c7a9fc7457/store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreITs.java#L33).

Reply:
This is because the `FederatedStoreITs` already has this pattern from its parent class `AbstractStoreITs`. This is not only redundant but means in debugging when you change the pattern in `AbstractStoreITs` to only run one IT you care about, the `FederatedStoreITs` still runs them all

Reply:
It is redundant, but I believe it was added to facilitate easier local testing of the federated store. Maybe we don't think that's needed. However, removing it is not really in scope for this PR/issue.